src/src/tx_main.cpp

```diff
@@ -502,6 +559,12 @@ void ICACHE_RAM_ATTR timerCallbackNormal()
     busyTransmitting = true;
     SendRCdataToRF();
   }
+  else
+  {
+  // End clear channel assessment by checking status
+  ChannelIsClear();
```

Comment:
Is this needed since it is called just before Radio.TXnb() ?

Reply:
removed this, not needed when enabling rx for lbt in tx interrupt.